api/features/views.py

```diff
@@ -108,6 +109,16 @@ def perform_create(self, serializer):
     def perform_update(self, serializer):
         serializer.save(project_id=self.kwargs.get("project_pk"))
 
+    def perform_destroy(self, instance):
+        message = FEATURE_DELETED_MESSAGE % self.name
+        AuditLog.objects.create(
```

Comment:
We should also include the environment here, right? We _could_ also include the `related_object_id` but I guess it won't be much use after it's deleted 🤔

Reply:
Feature Create/Delete works on a project level, right? So we can't add environment. Yeah, let's add ID (if maybe someone wants to keep track of it externally).

Reply:
One thing we can do is to create an audit log for every environment in that project because if environment is none, some integrations like slack(who works on environment level) will simply ignore this audit log.

Reply:
Sorry, yeah, you're right. Hmm... yeah, maybe we should create the AuditLog entry for feature deleted at the project level and then one for each of the environments? We can use `bulk_create` for that I guess?